composer.json

```diff
@@ -44,6 +44,11 @@
     "sort-packages": true,
     "platform": {
       "php": "7.3"
+    },
+    "allow-plugins": {
```

Comment:
Do we need these added 100%? Assuming you needed these to get your local up and running appropriately? They are fine to stay in im sure!

Reply:
Yeah these came along with having to update lando and composer so i think we're just a bit fresher with our versions and deps now!

Reply:
heck yes